e2e/rook-test-framework/objects/command_arg.go

```diff
@@ -0,0 +1,20 @@
+package objects
+
+
+type Command_Args struct {
+	Command             string
+	SubCommand		string
```

Comment:
this file is a good example where automatic formatting via `gofmt` or `goreturns` would be useful.  All the extra blank lines and uneven spacing would be fixed for you automatically.